### PR TITLE
Dilpreet | Test | Fixed all dialog close on route change 

### PR DESCRIPTION
**Pull request by dilpreet77**
https://app.clickup.com/t/201tx09

### CODE REVIEW

File: apps/web-giddh/src/app/inventory/components/new-branch-transfer/new.branch.transfer.list.component.ts [hunk 1]
```diff
@@ -12,7 +12,7 @@ import { InventoryService } from '../../../services/inventory.service';
 import { ReplaySubject, Observable, of as observableOf } from 'rxjs';
 import { Store, select } from '@ngrx/store';
 import { AppState } from '../../../store';
-import { takeUntil, take } from 'rxjs/operators';
+import { takeUntil, take, filter } from 'rxjs/operators';
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/inventory/components/new-branch-transfer/new.branch.transfer.list.component.ts:15]: Please check if RxJs `filter` operator is still required, if not, then remove the unused import

File: apps/web-giddh/src/app/inventory/components/new-branch-transfer/new.branch.transfer.list.component.ts [hunk 2]
```diff
@@ -149,6 +149,8 @@ export class NewBranchTransferListComponent implements OnInit, OnDestroy {
     }
 
     public ngOnInit(): void {
+
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/inventory/components/new-branch-transfer/new.branch.transfer.list.component.ts:152]: Avoid introduction of empty single line

File: apps/web-giddh/src/app/userDetails/components/subscriptions-plans/subscriptions-plans.component.ts [hunk 3]
```diff
@@ -1,4 +1,4 @@
-import { takeUntil } from 'rxjs/operators';
+import { filter, takeUntil } from 'rxjs/operators';
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/userDetails/components/subscriptions-plans/subscriptions-plans.component.ts:1]: Remove unsued imports

File: apps/web-giddh/src/app/userDetails/components/subscriptions-plans/subscriptions-plans.component.ts [hunk 4]
```diff
@@ -9,7 +9,7 @@ import { AuthenticationService } from '../../../services/authentication.service'
 import { AppState } from '../../../store';
 import { SettingsProfileActions } from '../../../actions/settings/profile/settings.profile.action';
 import { CompanyActions } from '../../../actions/company.actions';
-import { Router, ActivatedRoute } from '@angular/router';
+import { Router, ActivatedRoute, NavigationStart } from '@angular/router';
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/userDetails/components/subscriptions-plans/subscriptions-plans.component.ts:12]: Remove unsued imports

**Review by prateek-giddh — CHANGES_REQUESTED** [review]

**Review by prateek-giddh — APPROVED** [review]

**Review by ravinderwalkover — APPROVED** [review]

**Pull request merged**